pom.xml

```diff
@@ -138,6 +138,18 @@ flexible messaging model and an intuitive client API.</description>
         </exclusions>
       </dependency>
 
+      <dependency>
```

Comment:
This dependency should be added at the end of `all/src/assemble/LICENSE.bin.txt`

Reply:
I have added `org.apache.httpcomponents:httpclient` in client-shadeed pom and License file. However, I excluded all other jars from httpclient to avoid bringing lot of other things from it, but it requires `commons-logging:commons-logging` for internal logging without it we see ClassNotFoundException for logger class. So, I have added that dep explicitly and added to LICENSE and cliend-sahde as well.